### PR TITLE
name updates & fixes

### PR DESCRIPTION
**Pull request by cafemoloko**
FAQ and Calibration updates:
- product names
- content update
- links
- other fixes

### CODE REVIEW

File: source/pages/faq.rst [hunk 1]
```diff
@@ -578,7 +575,7 @@ How to enable depthai to perceive closer distances
 
 If the depth results for close-in objects look weird, this is likely because they are below the minimum depth-perception distance of DepthAI/OAK-D.
 
-For DepthAI Onboard Cameras (BW1098OBC) and OAK-D, the standard-settings minimum depth is around 70cm.
+For DepthAI Onboard Cameras (OAK-D-PCBA) and OAK-D, the standard-settings minimum depth is around 70cm.
```
Comment (VanDavv) [inline source/pages/faq.rst:578]:
```suggestion
For OAK-D, the standard-settings minimum depth is around 70cm.
```

File: source/pages/faq.rst [hunk 2]
```diff
@@ -588,7 +585,7 @@ Since the disparity-search of 96 is what limits the minimum depth, this means th
 
 2. Enable Extended Disparity.
 
-In Gen2, Extended Disparity is supported, which extends the disparity search to 192 pixels from the standard 96 pixels, thereby 1/2-ing the minimum depth, so making the minimum depth for BW1098OBC/OAK-D 35cm for 1280x800 resolution and around 19.6cm (limited by the focal distance of the grayscale cameras) for 640x400 resolution.
+In Gen2, Extended Disparity is supported, which extends the disparity search to 192 pixels from the standard 96 pixels, thereby 1/2-ing the minimum depth, so making the minimum depth for OAK-D-PCBA/OAK-D 35cm for 1280x800 resolution and around 19.6cm (limited by the focal distance of the grayscale cameras) for 640x400 resolution.
```
Comment (VanDavv) [inline source/pages/faq.rst:588]:
```suggestion
In Gen2, Extended Disparity is supported, which extends the disparity search to 192 pixels from the standard 96 pixels, thereby 1/2-ing the minimum depth, so making the minimum depth for OAK-D 35cm for 1280x800 resolution and around 19.6cm (limited by the focal distance of the grayscale cameras) for 640x400 resolution.
```

File: source/pages/faq.rst [hunk 3]
```diff
@@ -680,15 +677,15 @@ Monocular Neural Inference fused with Stereo Depth Mode
 
 For DepthAI units with onboard cameras, this works out to the following minimum depths:
 
-- DepthAI Raspberry Pi Compute Module Edition (`BW1097 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__) the minimum depth is **0.827** meters for full 1280x800 stereo resolution and  **0.414** meters for 640x400 stereo resolution:
+- OAK-D-CM3 (`OAK-D-CM3 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__) the minimum depth is **0.827** meters for full 1280x800 stereo resolution and  **0.414** meters for 640x400 stereo resolution:
```
Comment (VanDavv) [inline source/pages/faq.rst:680]:
```suggestion
- `OAK-D-CM4 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ the minimum depth is **0.827** meters for full 1280x800 stereo resolution and  **0.414** meters for 640x400 stereo resolution:
```

File: source/pages/faq.rst [hunk 4]
```diff
@@ -680,15 +677,15 @@ Monocular Neural Inference fused with Stereo Depth Mode
...
 
 .. code-block:: python
 
   min_distance = 857.06.15 * 0.09 / 96 = 0.803 # m
 
 calculation `here <https://www.google.com/search?safe=off&sxsrf=ALeKk00zuPUIqtKg9E4O1fSrB4IFp04AQw%3A1607995753791&ei=aRHYX57zL9P9-gTk5rmADA&q=857.06*.09%2F96&oq=857.06*.09%2F96&gs_lcp=CgZwc3ktYWIQAzIECCMQJ1CqJ1i8OmDlPGgAcAB4AIABX4gB9ASSAQE4mAEAoAEBqgEHZ3dzLXdpesABAQ&sclient=psy-ab&ved=0ahUKEwjey9H96s7tAhXTvp4KHWRzDsAQ4dUDCA0&uact=5>`__
 
-**OAK-D** and USB3C Onboard Camera Edition (`BW1098OBC <https://docs.luxonis.com/projects/hardware/en/latest/pages/DM1098OBC.html>`__) is
+**OAK-D** and USB3C Onboard Camera Edition (`OAK-D-PCBA <https://docs.luxonis.com/projects/hardware/en/latest/pages/DM1098OBC.html>`__) is
```
Comment (VanDavv) [inline source/pages/faq.rst:688]:
```suggestion
`OAK-D  <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1098OAK.html>`__ is
```

File: source/pages/faq.rst [hunk 5]
```diff
@@ -702,7 +699,7 @@ calculation `here <https://www.google.com/search?safe=off&sxsrf=ALeKk03HLvlfCWau
 Stereo Neural Inference Mode
 ----------------------------
 
-For DepthAI units with onboard cameras, all models (`BW1097 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ and `OAK-D <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1098OAK.html>`__) are
+For DepthAI units with onboard cameras, all models (`OAK-D-CM3 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ and `OAK-D <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1098OAK.html>`__) are
```
Comment (VanDavv) [inline source/pages/faq.rst:702]:
```suggestion
For DepthAI units with onboard cameras, all models (`OAK-D-CM4 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ and `OAK-D <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1098OAK.html>`__) are
```

File: source/pages/faq.rst [hunk 6]
```diff
@@ -734,13 +731,11 @@ For stereo baselines wider than 29cm, the minimum depth is limited by the horizo
...
+The :code:`extended disparity` mode affords a closer minimum distance for the given baseline. This increases the maximum disparity search from 96 to 192. So this cuts the minimum perceivable distance in half (given that the minimum distance is now :code:`focal_length * base_line_dist / 192` instead of :code:`focal_length * base_line_dist / 96`).
 
-- DepthAI Raspberry Pi Compute Module Edition (`BW1097 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__): **0.414** meters
-- OAK-D and USB3C Onboard Camera Edition (`OAK-D <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1098OAK.html>`__) is
-  **0.345** meters
-- Modular Cameras at Minimum Spacing (e.g. `DepthAI FFC <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1098FFC.html>`__) is
-  **0.115** meters
+- `OAK-D-CM3 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__): **0.414** meters
```
Comment (VanDavv) [inline source/pages/faq.rst:736]:
```suggestion
- `OAK-D-CM4 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__): **0.414** meters
```

File: source/pages/faq.rst [hunk 7]
```diff
@@ -777,12 +772,12 @@ So if the object detector is not the limit, the maximum distance will be limited
 
   Dm = (baseline/2) * tan_d((90 - HFOV / HPixels)*pi/2)
 
-For DepthAI HFOV = 73.5(+/-0.5) degrees, and HPixels = 1280.  And for the BW1098OBC, the baseline is 7.5cm.
+For DepthAI HFOV = 73.5(+/-0.5) degrees, and HPixels = 1280.  And for the OAK-D-PCBA, the baseline is 7.5cm.
```
Comment (VanDavv) [inline source/pages/faq.rst:775]:
```suggestion
For DepthAI HFOV = 73.5(+/-0.5) degrees, and HPixels = 1280.  And for the OAK-D, the baseline is 7.5cm.
```

File: source/pages/faq.rst [hunk 8]
```diff
@@ -777,12 +772,12 @@ So if the object detector is not the limit, the maximum distance will be limited
...
 
 So using this formula for existing models the *theoretical* max distance is:
 
-- BW1098OBC (OAK-D; 7.5cm baseline): 38.4 meters
-- `BW1097 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ (9cm baseline): 46 meters
+- OAK-D-PCBA (OAK-D; 7.5cm baseline): 38.4 meters
+- `OAK-D-CM3 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ (9cm baseline): 46 meters
```
Comment (VanDavv) [inline source/pages/faq.rst:780]:
```suggestion
- `OAK-D-CM4 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ (9cm baseline): 46 meters
```

File: source/pages/faq.rst [hunk 9]
```diff
@@ -832,9 +827,9 @@ DepthAI does convert to depth onboard for both the :code:`depth` stream and also
 
 But we also allow the actual disparity results to be retrieved so that if you would like to use the disparity map directly, you can.
 
-To calculate the depth map from the disparity map, it is (approximately) :code:`baseline * focal / disparity`.  Where the baseline is 7.5cm for BW1098OBC, 4.0cm for BW1092, and 9.0cm for `BW1097 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__, and the focal length is :code:`883.15` (:code:`focal_length = 1280/(2*tan(73.5/2/180*pi)) = 857.06`) for all current DepthAI models.
+To calculate the depth map from the disparity map, it is (approximately) :code:`baseline * focal / disparity`.  Where the baseline is 7.5cm for OAK-D-PCBA, 4.0cm for OAK-D-IoT-40, and 9.0cm for `OAK-D-CM3 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__, and the focal length is :code:`883.15` (:code:`focal_length = 1280/(2*tan(73.5/2/180*pi)) = 857.06`) for all current DepthAI models.
```
Comment (VanDavv) [inline source/pages/faq.rst:830]:
```suggestion
To calculate the depth map from the disparity map, it is (approximately) :code:`baseline * focal / disparity`.  Where the baseline is 7.5cm for OAK-D, 4.0cm for OAK-D-IoT-40, and 9.0cm for `OAK-D-CM4 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__, and the focal length is :code:`883.15` (:code:`focal_length = 1280/(2*tan(73.5/2/180*pi)) = 857.06`) for all current DepthAI models.
```

File: source/pages/faq.rst [hunk 10]
```diff
@@ -980,7 +975,7 @@ By default there are keyframes every 1 second which resolve the previous issues
 
 When running :code:`depthai_demo.py`, one can record a JPEG of the current frame by hitting :code:`c` on the keyboard.
 
-An example video encoded on DepthAI `BW1097 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ (Raspberry Pi Compute Module Edition) is below.  All DepthAI and megaAI units have the same 4K color camera, so will have equivalent performance to the video below.
+An example video encoded on DepthAI `OAK-D-CM3 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ (Raspberry Pi Compute Module Edition) is below.  All DepthAI and megaAI units have the same 4K color camera, so will have equivalent performance to the video below.
```
Comment (VanDavv) [inline source/pages/faq.rst:978]:
```suggestion
An example video encoded on DepthAI `OAK-D-CM4 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ (Raspberry Pi Compute Module Edition) is below.  All DepthAI and megaAI units have the same 4K color camera, so will have equivalent performance to the video below.
```

File: source/pages/faq.rst [hunk 11]
```diff
@@ -1191,10 +1186,10 @@ What Information is Stored on the DepthAI Boards
 ################################################
 
 Initial Crowd Supply backers received boards which had literally nothing stored on them.  All information was loaded
-from the host to the board.  This includes the BW1097 (`BW1097 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__), which had the calibration stored on the included microSD card.
+from the host to the board.  This includes the OAK-D-CM3 (`OAK-D-CM3 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__), which had the calibration stored on the included microSD card.
```
Comment (VanDavv) [inline source/pages/faq.rst:1189]:
```suggestion
from the host to the board.  This includes the `OAK-D-CM4 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__, which had the calibration stored on the included microSD card.
```

File: source/pages/faq.rst [hunk 12]
```diff
@@ -1191,10 +1186,10 @@ What Information is Stored on the DepthAI Boards
...
 
-So each hardware model which has stereo cameras (e.g. `BW1097 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__,
-`DepthAI FFC <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1098FFC.html>`__, `OAK-D <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1098OAK.html>`__, and
+So each hardware model which has stereo cameras (e.g. `OAK-D-CM3 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__,
```
Comment (VanDavv) [inline source/pages/faq.rst:1191]:
```suggestion
So each hardware model which has stereo cameras (e.g. `OAK-D-CM4 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__,
```

File: source/pages/faq.rst [hunk 13]
```diff
@@ -1283,7 +1278,7 @@ DepthAI and MegaAI are capable of both Gen1 and Gen2 USB3 - but not all USB3 hos
 
 DepthAI and megaAI can do h.264 and h.265 (HEVC) encoding on-device. The max resolution/rate is 4K at 30FPS.
 With the default encoding settings in DepthAI/megaAI, this brings the throughput down from 373MB/s (raw/unencoded 4K/30) to
-3.125MB/s (h.265/HEVC at 25mbps bit rate).  An example video encoded on DepthAI `BW1097 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ (Raspberry Pi Compute Module Edition) is below:
+3.125MB/s (h.265/HEVC at 25mbps bit rate).  An example video encoded on DepthAI `OAK-D-CM3 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ (Raspberry Pi Compute Module Edition) is below:
```
Comment (VanDavv) [inline source/pages/faq.rst:1281]:
```suggestion
3.125MB/s (h.265/HEVC at 25mbps bit rate).  An example video encoded on DepthAI `OAK-D-CM4 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ (Raspberry Pi Compute Module Edition) is below:
```

File: source/pages/faq.rst [hunk 14]
```diff
@@ -1630,19 +1625,19 @@ And note that we now have a specific ARMv6 (Pi Zero) specific build of DepthAI.
 How Much Power Does the DepthAI Raspberry Pi CME Consume?
 #########################################################
 
-The DepthAI Raspberry Pi Compute Module Edition (Raspberry Pi CME or `BW1097 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ for short) consumes around 2.5W idle and 5.5W to 6W when DepthAI is running full-out.
+The OAK-D-CM3 (Raspberry Pi CME or `OAK-D-CM3 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ for short) consumes around 2.5W idle and 5.5W to 6W when DepthAI is running full-out.
```
Comment (VanDavv) [inline source/pages/faq.rst:1628]:
```suggestion
The `OAK-D-CM4 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__ consumes around 2.5W idle and 5.5W to 6W when DepthAI is running full-out.
```

Comment (VanDavv) [reply source/pages/faq.rst:1628]: Is this correct power usage for CM4 too? Don't know if it changed between CM3 and CM4 (CC @Luxonis-David)

**Review by VanDavv — CHANGES_REQUESTED** [review]
I would change CM3 and OAK-D-PCBA mentions to CM4 and OAK-D respectively, I think these are more common

File: source/pages/faq.rst (hunk 14 above)
Comment (Luxonis-Brandon) [reply source/pages/faq.rst:1628]: Great question!

**Comment by Luxonis-Brandon** [comment]
Great ideas - thanks @VanDavv !

**Pull request merged**